corehq/apps/reports/tests/test_readable_formdata.py

```diff
@@ -283,8 +283,8 @@ def _test_corpus(self, slug):
         # Search for 'READABLE FORMS TEST' for more info
         # to bootstrap a test and have it print out your yaml result
         # uncomment this line. Unconventional, but it works.
-        # print yaml.safe_dump([json.loads(json.dumps(x.to_json()))
-        #                       for x in questions])
+        # print(yaml.safe_dump([json.loads(json.dumps(x.to_json()))
+        #                       for x in questions]))
```

Comment:
Would it be useful to include instructions for keeping the py3 tests in sync? I didn't know they existed or were different.

Reply:
https://github.com/dimagi/commcare-hq/pull/24294